lib/process-services-cloud/src/lib/process/process-filters/services/process-filter-cloud.service.spec.ts

```diff
@@ -236,4 +240,15 @@ describe('ProcessFilterCloudService', () => {
 
         expect(updatePreferenceSpy).toHaveBeenCalledWith('mock-appName', 'process-filters-mock-appName-mock-username', fakeProcessCloudFilters);
     });
+
+    it('should return engine event task subscription', (done) => {
+        spyOn(notificationCloudService, 'makeGQLQuery').and.returnValue(of(processCloudEngineEventsMock));
+
+        service.getProcessNotificationSubscription('testApp').subscribe((res) => {
```

Comment:
This is a suggestion, instead of having this observable with the subscription here and the done function to return the type wouldn't it be easier to have this transformed to a promise (toPromise) and await with the results?

Reply:
Even better to use `firstValueOF` await [firstValueFrom](https://rxjs.dev/api/index/function/firstValueFrom)(source$);